packages/design-picker/src/components/unified-design-picker.tsx

```diff
@@ -381,12 +382,19 @@ const StaticDesignPicker: React.FC< StaticDesignPickerProps > = ( {
 	purchasedThemes,
 } ) => {
 	const hasCategories = !! categorization?.categories.length;
+	const blankCanvasDesign = {
```

Comment:
Could we keep it the same as real data? thanks!

![image](https://user-images.githubusercontent.com/13596067/186833063-e27353ba-13b1-4dd3-b987-1b6414b065fa.png)

Reply:
Sure, where do you find it?

Reply:
I just modify the endpoint to let it return the blank canvas!

Reply:
Got it